apps/etterlatte-behandling-kafka/src/main/kotlin/no/nav/etterlatte/AppBuilder.kt

```diff
@@ -81,7 +81,7 @@ class AppBuilder(
         )
     }
 
-    private val behandlingApp: HttpClient by lazy {
+    private val behandlingAppExpectSuccess: HttpClient by lazy {
```

Comment:
Trenger vi to httpClients her? Kan ikke bare den eksisterende `private fun httpClient(scope: EnvEnum)` få inn et ekstra parameter `forventetSuksess` og brukes også for behandling?

Reply:
Regner med at dette er et retorisk spørsmål?

Reply:
Det kan settes på requesten, endret og bare en Klient nå som defaulter til success men skrudd av for det ENE ENDepuNKtet